src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -224,6 +272,11 @@ def load(
                 )  # Define profile name that will be merged from other layers
             profile_props = {**profile_loaded.data, **profile_props}
 
+            # Loading $schema property from project config
+            if config_type in ("Project Config"):
+                path_config_json = cfg._location + "/zowe.config.json"
+                schema_path = self.load_schema(cfg, path_config_json, opt_in)
```

Comment:
I don't know if it is require to store the output in some variable or just call it and do not return anything from the function

Reply:
I don't think we need to store the output in a variable in this file. It should be sufficient to have `schema_path` defined as a property of the ConfigFile class.

Can we validate the schema for all config layers rather than only for "Project Config"?